src/bot.ts

```diff
@@ -1177,6 +1192,143 @@ export class DiscordBot {
         await this.OnMessage(newMsg);
     }
 
+    public async OnMessageReactionAdd(reaction: Discord.MessageReaction, user: Discord.User | Discord.PartialUser) {
+        const message = reaction.message;
+        log.info(`Got message reaction add event for ${message.id} with ${reaction.emoji.name}`);
+
+        let rooms: string[];
+
+        try {
+            rooms = await this.channelSync.GetRoomIdsFromChannel(message.channel);
+
+            if (rooms === null) {
+                throw Error();
+            }
+        } catch (err) {
+            log.verbose("No bridged rooms to send message to. Oh well.");
+            MetricPeg.get.requestOutcome(message.id, true, "dropped");
+            return;
+        }
+
+        const intent = this.GetIntentFromDiscordMember(user);
+        await intent.ensureRegistered();
+        this.userActivity.updateUserActivity(intent.userId);
+
+        const storeEvent = await this.store.Get(DbEvent, {
+            discord_id: message.id
+        });
+
+        if (!storeEvent?.Result) {
+            return;
```

Comment:
Rather than saying the message doesn't exist, maybe it's better to say it's untracked?

When the logs are separated from the code (which is how I'd expect it to be consumed quite often, especially for bug reports etc) readers may think it means the Discord message doesn't exist.

The term "untracked" makes clear it's just a message the bridge wasn't aware of, or perhaps wasn't online to bridge at the time the original message the reaction was applied to was sent.

i.e.

```ts
log.verbose(`Received reaction event to untracked message. Dropping! Reaction Event: ${reaction}`);
```